docs/src/pages/[platform]/theming/style-props/index.page.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: Style Props
+metaDescription: 'CSS Style Props that are available on every Amplify UI component'
+supportedFrameworks: react
+---
+
+import { Example, ExampleCode } from '@/components/Example';
+import { ComponentPropsToStylePropsMap, Card } from '@aws-amplify/ui-react';
+import {
+  BackgroundStyles,
+  BackgroundStylePropExample,
+  BorderStyles,
+  BorderStylePropExample,
+  ColorStyles,
+  ColorStylePropExample,
+  FlexStyles,
+  FlexStylePropExample,
+  GridStyles,
+  GridStylePropExample,
+  MarginAndPaddingStyles,
+  MarginAndPaddingStylePropExample,
+  PositionStyles,
+  PositionStylePropExample,
+  ShadowStyles,
+  ShadowStylePropExample,
+  SizeStyles,
+  SizeStylePropExample,
+  TypographyStyles,
+  TypographyStylePropExample,
+  OtherStyles,
+} from './examples';
+
+These style props are available on Amplify UI primitive components and can be used to modify the corresponding styles. This allows users to quickly make style adjustments without having to go through css or add a theme object. Style props override any conflicting css style sheet or theme styling.
+
```

Comment:
Before we get to the examples, perhaps we can briefly explain the three primary ways of styling. Something like this:
```
There are three ways of passing style props to a component:

1. Pass any valid CSS property directly (e.g., `backgroundColor="red"`)
- This is useful for rapid prototyping or styling a component which differs from the main theme

2. Use a design token from the Theme object by destructuring `tokens` from the `useTheme` hook (e.g., `backgroundColor={tokens.colors.red[40]}`
- (I need some help explaining what a good use case for this would be @dbanksdesign @reesscot )

3. Use the shorthand syntax to reference design tokens without using the `useTheme` hook (e.g., `backgroundColor="red.40"`)
- (I need some help explaining how to figure out the shorthand syntax for any design token @zchenwei )
```

Reply:
Thanks added the bullet points listed please validate the wording.